projects/demo/src/polyfills.ts

```diff
@@ -1,20 +1,19 @@
 // IE9, IE10 and IE11 requires all of the following polyfills.
```

Comment:
think about this: 
The Angular team is deprecating support for Internet Explorer 11 in Angular v12 (to be released in May 2021 and supported through November 2022), and plans to remove support for this browser in Angular v13 (late 2021).